frc2357/robotlib2357
# Move ButtonChordTrigger to new source tree

This moves ButtonChordTrigger to the new source tree and adds tests for it.
This is a first example of how to move code over to the new library tree and add tests as we do it.

To Test:
1. Run "WPILib: Build Robot Code" and ensure success
2. Run "WPILib: Test Robot Code" and ensure success
